architecture/additional_concepts/storage.adoc

```diff
@@ -152,12 +152,11 @@ status of the volume.
 
 === Types of Persistent Volumes
 
-{product-title} supports the following `*PersistentVolume*` plug-ins:
+{product-title} supports the following `PersistentVolume` plug-ins:
 
 ifdef::openshift-enterprise,openshift-origin[]
 - xref:../../install_config/persistent_storage/persistent_storage_nfs.adoc#install-config-persistent-storage-persistent-storage-nfs[NFS]
 - xref:../../rest_api/kubernetes_v1.adoc#v1-hostpathvolumesource[HostPath]
-(single node testing only)
```

Comment:
@markturansky Bump. Pinging you since it looks like you introduced this line into the docs. Thanks!